src/script/script.h

```diff
@@ -11,12 +11,15 @@
 #include <assert.h>
 #include <climits>
 #include <limits>
+#include <map>
 #include <stdexcept>
 #include <stdint.h>
 #include <string.h>
 #include <string>
 #include <vector>
 
+#include <boost/assign/list_of.hpp>
```

Comment:
You might be right. I'll double check. I might have needed it at one point and then moved things around. If it's only needed in the cpp, I'll move it.